server/azure-appinsights.js

```diff
@@ -1,15 +1,16 @@
 const appInsights = require('applicationinsights')
-const fs = require('fs')
-const config = require('./config')
+const applicationVersion = require('./application-version')
 
-module.exports = () => {
```

Comment:
The issue was that the config is hidden behind a function that never gets executed.